db/migrations/20221115104182_add_index_to_service_instances_service_plan_id.rb

```diff
@@ -0,0 +1,7 @@
+Sequel.migration do
+  change do
+    alter_table :service_instances do
+      add_index :service_plan_id, name: :service_instances_service_plan_id_index
```

Comment:
I had a look at the other indexes for the `service_instances` table. As this new index name is quite long, we should use `si_service_plan_id_index` instead, similar to e.g.`si_created_at_index`.